src/components/SortableTable/index.tsx

```diff
@@ -119,13 +119,27 @@ export const SortableTable = <DataItem,>({
             </tr>
           </thead>
           <tbody>
-            {localData[currentPage - 1].map((item, index) => (
-              // eslint-disable-next-line react/no-array-index-key
-              <tr key={`row-${index}`}>
-                {hideNumberColumn ? null : <td>{index + 1}</td>}
-                {renderItemRow(item)}
-              </tr>
-            ))}
+            {localData[currentPage - 1].map((item, index) => {
+              const RowContents = () => (
+                <>
+                  {hideNumberColumn ? null : <td>{index + 1}</td>}
+                  {renderItemRow(item)}
+                </>
+              );
+
+              return (
+                // eslint-disable-next-line react/no-array-index-key
+                <tr key={`row-${index}`}>
+                  {item.href ? (
```

Comment:
thoughts on this API?

The idea is if DataItem contains a magic prop called `href`, it'll wrap the whole row in an anchor.

My thought is a) this is something we'll likely need frequently in tables and b) best to use an anchor to so users/screen readers can see where it links to

Reply:
I like this approach. The only change I would make is maybe adding `onClick` event on the anchor. Right now it's not using the router but does the full page refresh when going to the link and back. What do you think?

Reply:
Ah good cool - i should actual be using the router's `Link` component here